apps/designer/package.json

```diff
@@ -6,7 +6,7 @@
     "build": "remix build",
     "dev": "remix dev",
     "start": "remix-serve build",
-    "typecheck": "tsc -b",
+    "typecheck": "tsc",
```

Comment:
nice, I've noticed that this was a leftover here but had no idea that it could impact the watch behaviour like this